requirements/edx/base.txt

```diff
@@ -106,7 +106,7 @@ edx-milestones==0.3.0     # via -r requirements/edx/base.in
 edx-opaque-keys[django]==2.2.0  # via -r requirements/edx/paver.txt, edx-bulk-grades, edx-ccx-keys, edx-completion, edx-drf-extensions, edx-enterprise, edx-milestones, edx-organizations, edx-proctoring, edx-user-state-client, edx-when, lti-consumer-xblock, xmodule
 edx-organizations==6.9.0  # via -r requirements/edx/base.in
 edx-proctoring-proctortrack==1.0.5  # via -r requirements/edx/base.in
-edx-proctoring==3.6.0     # via -r requirements/edx/base.in, edx-proctoring-proctortrack
+edx-proctoring==3.5.0     # via -c requirements/edx/../constraints.txt, -r requirements/edx/base.in, edx-proctoring-proctortrack
```

Comment:
It was pinned yesterday because `edx-proctoring>3.5.0` caused test failures.

Reply:
there are no test failures currently and it seems to be using 3.6